plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/controller/UserController.java

```diff
@@ -152,7 +152,7 @@ public Object updateUser(@RequestParam String username, @RequestParam String new
                 response.sendError(HttpServletResponse.SC_FORBIDDEN, "authorization failed!");
                 return null;
             }
-        } catch (HttpSessionRequiredException e) {
+        } catch (HttpSessionRequiredException | AccessException e) {
```

Comment:
AccessException should be `authorization failed!`?

Reply:
Yes, I'll reorganize it